main.go

```diff
@@ -98,13 +98,15 @@ func main() {
 		probeAddr            string
 		pprofAddr            string
 		enableLeaderElection bool
+		createTreeDeadline   int64
 	)
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&pprofAddr, "pprof-address", "", "The address to expose the pprof server. Default is empty string which disables the pprof server.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.Int64Var(&createTreeDeadline, "create-tree-deadline", 1200, "The time allowance for the create tree job to run before failing.")
```

Comment:
Perhaps mention that the value is seconds to make that clear.

Something like this:
```suggestion
	flag.Int64Var(&createTreeDeadline, "create-tree-deadline", 1200, "The time allowance (in seconds) for the create tree job to run before failing.")
```